pt/lessons/specifics/ecto.md

```diff
@@ -124,25 +112,19 @@ defmodule ExampleApp.Repo.Migrations.CreateUser do
 end
 ```
 
-Por padrão Ecto cria uma chave primária `id` auto incrementado. Aqui estamos a usar
-o callback padrão `change/0` mas Ecto também suporta `up/0` e `down/0` no caso de
-precisar um controle mais granular.
+Por padrão Ecto cria uma chave primária `id` auto incrementado. Aqui estamos a usar o callback padrão `change/0` mas Ecto também suporta `up/0` e `down/0` no caso de precisar um controle mais granular.
 
-Como você deve ter adivinhado, adicionando `timestamps` na sua migration irá criar
-e gerir os campos `inserted_at` e `updated_at` por você.
+Como você deve ter adivinhado, adicionando `timestamps` na sua migration irá criar e gerir os campos `inserted_at` e `updated_at` por você.
 
-Para aplicar as alterações definidas na nossa migration, roda `mix ecto.migrate`.
+Para aplicar as alterações definidas na nossa migration, rode `mix ecto.migrate`.
 
-Para mais informações dê uma olhada a secção [Ecto.Migration](http://hexdocs.pm/ecto/Ecto.Migration.html#content)
-da documentação.
+Para mais informações dê uma olhada a seção [Ecto.Migration](http://hexdocs.pm/ecto/Ecto.Migration.html#content) da documentação.
 
-## Modelos
+## Schemas
 
-Agora que temos nossa migration podemos continuar para o modelo. Modelos definem o
-nosso esquema, funções auxiliares, e nosso *changeset*. Iremos falar mais sobre
-*changesets* nas próximas secções.
+Agora que temos nossa migration podemos continuar para o schemas. Schema é um módulo, que define define mapeamentos para uma tabela do banco de dados e seus campos, funções auxiliares, e nossos *changesets*. Iremos falar mais sobre *changesets* nas próximas seções.
```

Comment:
Small typo: "... continuar para o schem**a**.".
Also, "... que define mapeamentos ..." has the "define" word twice.